lib/commands/start.mjs

```diff
@@ -152,3 +178,4 @@ async function getArtifactStatus(apiToken, artifactDir, sha) {
     }
 }
 
+}
```

Comment:
Add a single carriage return at the end.

Reply:
Eric, what is a single carriage return? I looked it up and the explanations did not make it obvious what you wanted. 

Did you want to eliminate the empty line in line 180, or something else?